src/excalidraw-app/collab/Portal.tsx

```diff
@@ -32,6 +33,7 @@ class Portal {
     this.socket.on("init-room", () => {
       if (this.socket) {
         this.socket.emit("join-room", this.roomId);
+        trackEvent("share", "room joined");
```

Comment:
Just want to make sure the intention here is to track it for all types of join — in other words, this will log also when the room is created manually. If that's intended, we can merge.